kclvm/driver/src/test_data/test_vendor/.kpm/config/kpm.json

```diff
@@ -0,0 +1 @@
+{"DefaultOciRegistry":"ghcr.io","DefaultOciRepo":"kcl-lang","DefaultOciPlainHttp":false}
```

Comment:
Should it be removed after testing is completed?

Reply:
fixed.